src/SIM/AdaptiveSetup.h

```diff
@@ -54,11 +54,13 @@ class AdaptiveSetup
   //! \param[in] iStep Refinement step counter
   //! \param[in] gNorm Global norms
   //! \param[in] refIn Element refinement indicators (element error norms)
+  //! \param[in] currDofs Current number of model DOFs. -1 to use model.getNoDOFs()
   //! \return Number of elements to be refined
   //! \return If zero, no refinement needed
   //! \return Negative value on error
   int calcRefinement(LR::RefineData& prm, int iStep,
-                     const Vectors& gNorm, const Vector& refIn) const;
+                     const Vectors& gNorm, const Vector& refIn,
+                     const int currDofs = -1) const;
```

Comment:
Remove `const` here. Mismatch with definition file and the value may be changed there.

Reply:
yeah. fixed the source, forgot the header..